frontend/pages/users.tsx

```diff
@@ -46,7 +46,7 @@ const Users: NextPage = () => {
    *
    * {@label USERS_PAGE_ERROR}
    */
-  const [error, setError] = useState('');
+  const [error, setError] = useState('This is an error');
```

Comment:
This is helpfull to check if it works. But this probably shouldn't be added.

Reply:
Ah yeah right, forgot about that hahaha